extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/config/GrpcClientConfiguration.java

```diff
@@ -143,4 +143,10 @@
      */
     @ConfigItem(defaultValue = "pick_first")
     public String loadBalancingPolicy;
+
+    /**
+     * The compression to use for each call.
```

Comment:
Can you list the supported values?
Also, I believe we only support gzip in native at the moment (I should be able to get snappy to work too as I did it for Kafka).

Reply:
It seems that the compressor must be registered in the current `CompressorRegistry` and by default only `gzip` is registered.  And I have no idea if we support a custom `CompressorRegistry` in quarkus.